app/src/Routes/Works/works.json

```diff
@@ -770,9 +712,9 @@
         "attribution": "Recorded by Andrew Gignac, trumpet",
         "movements": [],
         "duration": "5 minutes",
-        "publisher": "Hal Leonard - Southern Music",
+        "publisher": "Keiser Southern Music; distributed by Hal Leonard",
         "publisherLinks": [
-          "https://www.halleonard.com/"
+          "HL# 3776282 - https://www.halleonard.com/search/search.action?keywords=3776282"
```

Comment:
The links should just be the link by itself, we should find another way of displaying custom text for the link